appveyor.yml

```diff
@@ -121,29 +94,41 @@ install:
   ## Install pip - disabled because pip is already installed
   #- ps: (new-object System.Net.WebClient).Downloadfile('https://bootstrap.pypa.io/get-pip.py', 'C:\Users\appveyor\get-pip.py')
   #- ps: Start-Process -FilePath "C:\Python27\python.exe" -ArgumentList "C:\Users\appveyor\get-pip.py" -Wait -Passthru
+  - where pip
+  - pip --version
+
+  # Verify that Chocolatey is available
+  - where choco
+  - choco --version
+
+  # Circumvention:
+  # The installation in pywbem_os_setup.bat installs swig but without generating
+  # the swig.exe shim file (using GenShim).
+  # See GenShim issue https://github.com/chocolatey/shimgen/issues/43
+  # Therefore, we install swig ihere in order to create the swig.exe shim file.
```

Comment:
What is word ihere??

Reply:
"here" with an "i" for vi insert mode ;-)
DONE.